### DISCUSSION TITLE
Is this streamable?

### DISCUSSION THREAD

**Discussion by SeaWolfX** [chromiecraft/chromiecraft#5722] [General]
My wife and I stream some games on Twitch (mostly as a form of therapy for her plethora of mental disorders). Is it safe to stream playing this server there? I've just got her interested in playing and we rolled some new guys to level together. It'd be awesome if we could cause 80% of our weekly gaming time is used streaming. I've done some light google research on the subject but came across a lot of grey answers. Wasn't sure if someone here would have a more definitive yes or no.

**Comment by BeefSteaker** [comment]
I believe it does break twitch TOS and if blizzard cared enough you'd get banned. But plenty of people stream private servers so just don't be too blatant about it

**Reply by SeaWolfX** [reply]
The more I've read the more that's what I've gathered. We get 30s viewcount on good nights so I doubt they'll notice us, and it does look like we'd get a warning at least before twitch will ban us as well
